cocotb/__init__.py

```diff
@@ -128,6 +128,14 @@ def _reopen_stream_with_buffering(stream_name):
 _library_coverage = None
 """ used for cocotb library coverage """
 
+top = None  # type: SimHandleBase
+"""
+A handle to the :make:var:`TOPLEVEL` entity. ``None`` if :mod:`cocotb` was not loaded from a simulator.
+This is equivalent to the :term:`DUT` parameter given to cocotb tests, so it can be used wherever that variable can be used.
+It is particularly useful for extracting information about the :term:`DUT` in module-level class and function definitions;
+and in parameters to :class:`~cocotb.regression.TestFactory`s.
```

Comment:
`TestFactory` isn't displaying properly in the docs. This is what the examples page uses:
```suggestion
and in parameters to :class:`.TestFactory`s.
```

Reply:
Lets try it. I am not sure why that works. The relative path implies the current module is `cocotb.regression`, but I don't see that being set anywhere.

Reply:
Maybe this works?
```
:class:`TestFactories <cocotb.regression.TestFactory>`
```

Reply:
The problem is likely the "s" appended directly to the ReST markup which breaks the parsing.

Reply:
You can use `` :class:`.TestFactory`\ s`` to fix that